src/yunohost/service.py

```diff
@@ -498,11 +511,11 @@ def _run_service_command(action, service):
     if service not in services.keys():
         raise MoulinetteError(errno.EINVAL, m18n.n('service_unknown', service=service))
 
-    cmd = None
-    if action in ['start', 'stop', 'restart', 'reload', 'enable', 'disable']:
-        cmd = 'systemctl %s %s' % (action, service)
-    else:
-        raise ValueError("Unknown action '%s'" % action)
+    possible_actions = ['start', 'stop', 'restart', 'reload', 'enable', 'disable']
+    if action not in possible_actions:
+        raise ValueError("Unknown action '%s', available actions are: %s" % (action, ", ".join(possible_actions)))
```

Comment:
Maybe an assert here would be slightly more elegant, but that's already pretty better than before :+1: